stage1/init/kvm/network.go

```diff
@@ -158,7 +158,9 @@ func GenerateNetworkInterfaceUnits(unitsPath string, netDescriptions []netDescri
 			return errwrap.Wrap(fmt.Errorf("failed to create network unit file %q", unitName), err)
 		}
 
-		rlog.Printf("network unit created: %q in %q (iface=%q, addr=%q)", unitName, unitsPath, ifName, address)
+		if debug {
```

Comment:
You're right. It does not change things. However, I gave this patch a quick try and had the same experience. It seems the debug variable is not getting passed through at some point. I think if that gets fixed using SetDebug should also work.